### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by mre**

### CODE REVIEW

File: content/podcast/s02e03-thunderbird/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+season = "02"
+episode = "03"
+series = "Podcast"
++++
+
+There are probably only a handful of open-source projects that had a bigger
+impact on the world than Mozilla Thunderbird. The email client has been around
+for over two decades and has been a staple for many users (me included).
+
+Dealing with a legacy codebase that servers millions of users is no easy feat.
```
Comment (M3t0r) [inline content/podcast/s02e03-thunderbird/index.md]:
```suggestion
Dealing with a legacy codebase that serves millions of users is no easy feat.
```

File: content/podcast/s02e03-thunderbird/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+### About Brendan Abolivier
+
+Brendan joined the Thunderbird team in 2023 and has been working on the Rust-based Exchange protocol support.
+Previously, he worked on the Matrix protocol and the Element chat client.
+He is a strong advocate for open-source software and maintains projects written in Rust, Python, and Go.
+
+### Links From The Show
+
+- [Bringing Exchange Support To Thunderbird](https://blog.thunderbird.net/2024/04/adventures-in-rust-bringing-exchange-support-to-thunderbird/)
+- [FOSDEM 2024: How to Exchange Rot For Rust](https://fosdem.org/2024/schedule/event/fosdem-2024-2469-thunderbird-how-to-exchange-rot-for-rust/)
```
Comment (M3t0r) [inline content/podcast/s02e03-thunderbird/index.md]:
```suggestion
- [Bringing Exchange Support To Thunderbird](https://blog.thunderbird.net/2024/04/adventures-in-rust-bringing-exchange-support-to-thunderbird/)
- [element](https://element.io/)
- [State of Open conference London](https://stateofopencon.com/)
- [Thunderbird as part of MZLA](https://blog.thunderbird.net/2020/01/thunderbirds-new-home/)
- [Evolution Exchange Support](https://help.gnome.org/users/evolution/2.32/features.html.en)
- [FOSDEM'24: "Thunderbird: How to Exchange Rot For Rust"](https://fosdem.org/2024/schedule/event/fosdem-2024-2469-thunderbird-how-to-exchange-rot-for-rust/)
- [quick_xml crate](https://github.com/tafia/quick-xml)
- [http crate](https://github.com/hyperium/http)
```

I don't quite know where the "Bringing Exchange Support To Thunderbird" blog posts fits in, it isn't mentioned in the episode, but I think at the very beginning isn't a bad idea.

**Review by M3t0r — CHANGES_REQUESTED** [review]

**Comment by mre** [comment]
Thanks for the review @M3t0r.

File: content/podcast/s02e03-thunderbird/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+### Links From The Show
+
+- [Bringing Exchange Support To Thunderbird](https://blog.thunderbird.net/2024/04/adventures-in-rust-bringing-exchange-support-to-thunderbird/)
+- [element](https://element.io/)
+- [State of Open conference London](https://stateofopencon.com/)
+- [Thunderbird as part of MZLA](https://blog.thunderbird.net/2020/01/thunderbirds-new-home/)
+- [Evolution Exchange Support](https://help.gnome.org/users/evolution/2.32/features.html.en)
+- [FOSDEM'24: "Thunderbird: How to Exchange Rot For Rust"](https://fosdem.org/2024/schedule/event/fosdem-2024-2469-thunderbird-how-to-exchange-rot-for-rust/)
+- [quick_xml crate](https://github.com/tafia/quick-xml)
+- [http crate](https://github.com/hyperium/http)
```
Comment (M3t0r) [inline content/podcast/s02e03-thunderbird/index.md]:
```suggestion
- [Bringing Exchange Support To Thunderbird](https://blog.thunderbird.net/2024/04/adventures-in-rust-bringing-exchange-support-to-thunderbird/)
- [element](https://element.io/)
- [synapse](https://github.com/element-hq/synapse)
- [State of Open conference London](https://stateofopencon.com/)
- [Thunderbird as part of MZLA](https://blog.thunderbird.net/2020/01/thunderbirds-new-home/)
- [Evolution Exchange Support](https://help.gnome.org/users/evolution/2.32/features.html.en)
- [FOSDEM'24: "Thunderbird: How to Exchange Rot For Rust"](https://fosdem.org/2024/schedule/event/fosdem-2024-2469-thunderbird-how-to-exchange-rot-for-rust/)
- [quick_xml crate](https://github.com/tafia/quick-xml)
- [http crate](https://github.com/hyperium/http)
- [Donate to Thunderbird](https://www.thunderbird.net/donate/)
```

File: content/podcast/s02e03-thunderbird/index.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+template = "episode.html"
+draft = false
+[extra]
+guest = "Brendan Abolivier"
+role = "Software Engineer"
+season = "02"
+episode = "03"
+series = "Podcast"
++++
+
```
Comment (M3t0r) [inline content/podcast/s02e03-thunderbird/index.md:13]:
```suggestion

<div><script id="letscast-player-d02c0302" src="https://letscast.fm/podcasts/rust-in-production-82281512/episodes/rust-in-production-ep-10-thunderbird-s-brendan-abolivier/player.js?size=m-alternative"></script></div>

```

**Review by M3t0r — CHANGES_REQUESTED** [review]

**Comment by mre** [comment]
@prscheduler 30/05/2024T16:00

**Review by M3t0r — APPROVED** [review]

**Pull request merged**